### PR TITLE
Added apigee-x-foundations blueprint

### PR DESCRIPTION
**Pull request by apichick**
Added apigee-x-foundations blueprint

---
**Checklist**

I applicable, I acknowledge that I have:
- [X] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [X] Ran `terraform fmt` on all modified files
- [X] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [X] Made sure all relevant tests pass

### CODE REVIEW

File: blueprints/apigee/apigee-x-foundations/README.md [hunk 1]
```diff
@@ -0,0 +1,469 @@
+# Apigee Foundations
+
+This module creates all the resources necessary to set up Apigee X on Google Cloud. Find below a few examples of different Apigee architectures that can be created using this module.
+
+Apigee can be exposed to clients using Regional Internal Application Load Balancer, Global External Application Load Balancer or both. When using the Regional Internal Application Load Balancer, self-managed certificates (incuding self-signed certificates generated in this same module) can be used. When using the Global External Application Load Balancer Google-managed certificates or self-managed certificates (including self-signed certificates generated in this same module).
```
Comment (ludoo) [inline blueprints/apigee/apigee-x-foundations/README.md:5]: "When using the Regional Internal Application Load Balancer,  [use] self-managed"

File: blueprints/apigee/apigee-x-foundations/README.md [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+
```
Comment (ludoo) [inline blueprints/apigee/apigee-x-foundations/README.md:6]: Can you add a TOC here? And then I would make section headings below shorter, maybe by abbreviating the LB names?

**Review by ludoo — APPROVED** [review]

File: blueprints/apigee/apigee-x-foundations/README.md (hunk 1 above)
Comment (apichick) [reply blueprints/apigee/apigee-x-foundations/README.md]: done

File: blueprints/apigee/apigee-x-foundations/README.md (hunk 2 above)
Comment (apichick) [reply blueprints/apigee/apigee-x-foundations/README.md]: done

**Pull request merged**